### PR TITLE
feat(v4-examples): migrate router provider examples starts with "a"

### PR DESCRIPTION
**Pull request by alicanerdurmaz**
- auth-antd
- auth-auth0
- auth-chakra-ui
- auth-google-login
- auth-headless
- auth-keycloak
- auth-mantine
- auth-mui
- auth-otp
- access-control-cerbos
- access-control-casbin
- antd-audit-log
- audit-log-provider

-----

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

File: examples/access-control-cerbos/src/App.tsx [hunk 1]
```diff
@@ -32,62 +33,104 @@ const cerbos = new Cerbos("https://demo-pdp.cerbos.cloud", {
...
+                            element={<CategoryShow />}
+                        />
+                        <Route
+                            path="/posts/create"
+                            element={<CategoryCreate />}
+                        />
+                        <Route
+                            path="/posts/edit/:id"
+                            element={<CategoryEdit />}
+                        />
```
Comment (yildirayunlu) [inline examples/access-control-cerbos/src/App.tsx:127]:
```suggestion
                        <Route path="/categories" element={<CategoryList />} />
                        <Route
                            path="/categories/show/:id"
                            element={<CategoryShow />}
                        />
                        <Route
                            path="/categories/create"
                            element={<CategoryCreate />}
                        />
                        <Route
                            path="/categories/edit/:id"
                            element={<CategoryEdit />}
                        />
```

File: examples/access-control-casbin/src/App.tsx [hunk 2]
```diff
@@ -22,68 +23,110 @@ const App: React.FC = () => {
...
+                        <Route path="/posts/show/:id" element={<PostShow />} />
+                        <Route path="/posts/create" element={<PostCreate />} />
+                        <Route path="/posts/edit/:id" element={<PostEdit />} />
+
+                        <Route path="/users" element={<UserList />} />
+                        <Route path="/users/show/:id" element={<UserShow />} />
+                        <Route path="/users/create" element={<UserCreate />} />
+                        <Route path="/users/edit/:id" element={<UserEdit />} />
+
+                        <Route path="/posts" element={<CategoryList />} />
```
Comment (BatuhanW) [inline examples/access-control-casbin/src/App.tsx:111]:
```suggestion
                        <Route path="/categories" element={<CategoryList />} />
```

File: examples/access-control-casbin/src/App.tsx [hunk 3]
```diff
@@ -22,68 +23,110 @@ const App: React.FC = () => {
...
+                        <Route
+                            path="/posts/show/:id"
```
Comment (BatuhanW) [inline examples/access-control-casbin/src/App.tsx:113]:
```suggestion
                            path="/categories/show/:id"
```

**Review by BatuhanW — CHANGES_REQUESTED** [review]

File: examples/audit-log-provider/src/App.tsx [hunk 4]
```diff
@@ -1,61 +1,81 @@
...
+                        create: "/posts/create",
+                        edit: "/posts/edit/:id",
+                        meta: {
+                            canDelete: true,
+                        },
+                    },
+                ]}
+            >
+                <Routes>
+                    <Route element={<Outlet />}>
```
Comment (salihozdemir) [inline examples/audit-log-provider/src/App.tsx:64]: No need to add parent `Route` in here.
```suggestion
```

**Review by salihozdemir — APPROVED** [review]

**Review by salihozdemir — APPROVED** [review]

**Review by BatuhanW — APPROVED** [review]

**Review by yildirayunlu — APPROVED** [review]

**Pull request merged**